easybuild/easyconfigs/e/elbencho/elbencho-2.2-5-GCC-11.3.0.eb

```diff
@@ -0,0 +1,37 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Maicon Faria <maicon.faria@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'MakeCp'
+
+name = 'elbencho'
+version = '2.2-5'
+
+homepage = 'https://github.com/breuner/elbencho'
+description = """ A distributed storage benchmark for files, objects & blocks with support for GPUs"""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/breuner/elbencho/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['4b598639452665a8b79c4c9d8a22ae63fb9b04057635a45e686aa3939ee255b4']
+
+dependencies = [
+    ('CUDA', '12.0.0', '', True),
```

Comment:
If `CUDA` is included as dependency, that should be reflected in the `versionsuffix`:

```python
versionsuffix = '-CUDA-%(cudaver)s`
```

I'm surprised our tests don't trip over this... 🤔